chat.js

```diff
@@ -951,14 +950,14 @@ Chat.uncacheTree = function (root) {
 	let uncache = [require.resolve(root)];
 	do {
 		let newuncache = [];
-		for (let i = 0; i < uncache.length; ++i) {
-			if (require.cache[uncache[i]]) {
+		for (const i of uncache) {
```

Comment:
This isn't equivalent

Reply:
Should it just be reverted to original? Not sure how the uncachething works